public/docs/dart/latest/guide/server-communication.jade

```diff
@@ -5,11 +5,11 @@ block includes
   - var _Http = 'BrowserClient';
```

Comment:
This results in a lot of "BrowserClient client" occurrences in the page, which is awkward but OK for now.